Library/Formula/saltstack.rb

```diff
@@ -9,80 +10,96 @@ class Saltstack < Formula
     sha1 "738d8efecb7b916f10acf8707822945623226524" => :mountain_lion
   end
 
-  head "https://github.com/saltstack/salt.git", :branch => "develop", :shallow => false
+  devel do
+    url "https://github.com/saltstack/salt/archive/v2015.2.0rc2.tar.gz"
+    sha256 "be71c1f2f9f878d5f958396620983c5981f55eaf32913e7f28c129c35f37657a"
+    version "2015.2.0rc2"
+  end
 
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "swig" => :build
   depends_on "zeromq"
   depends_on "libyaml"
 
+  # For vendored Swig
+  depends_on "pcre"
```

Comment:
This may actually only be a ` build ` dependency given ` swig ` is only a build dependency as well.